### PR TITLE
Fixes #9892: Remove relay packages on old systems and add server packages to sles 12

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9892

### CODE REVIEW

File: release-data/changelogs/changelog-4.1.md [hunk 1]
```diff
@@ -19,12 +19,15 @@ a quick and easy way to get an installation for testing.
...
+  - Rudder server and Rudder relay: **Debian 7, Debian 8, RHEL/CentOS 6, RHEL/CentOS 7
+    (64 bits), SLES 11, SLES 12, Ubuntu 14.04, Ubuntu 16.04**
   - Rudder agent: all of the above plus **Debian 5, Debian 6,
     RHEL/CentOS 3, RHEL/CentOS 5, CentOS 7 (32 bits), Fedora 18, SLES
     10, SLES 12, Ubuntu 10.04, Ubuntu 12.04, Ubuntu 12.10**
   - Rudder agent (binary packages available from
     ([Normation](http://www.normation.com)): **Windows Server 2008-2012,
     AIX 5-6-7**
 
+**Note** : rudder-agent-thin and rudder-relay have been removed from Debian 5, Debian 6,
```
Comment (amousset) [inline release-data/changelogs/changelog-4.1.md]: the package is called rudder-server-relay

**Comment by peckpeck** [comment]
Commit modified

File: release-data/changelogs/changelog-4.1.md [hunk 2]
```diff
@@ -19,12 +19,15 @@ a quick and easy way to get an installation for testing.
...
+  - Rudder server and Rudder relay: **Debian 7, Debian 8, RHEL/CentOS 6, RHEL/CentOS 7
+    (64 bits), SLES 11, SLES 12, Ubuntu 14.04, Ubuntu 16.04**
   - Rudder agent: all of the above plus **Debian 5, Debian 6,
     RHEL/CentOS 3, RHEL/CentOS 5, CentOS 7 (32 bits), Fedora 18, SLES
     10, SLES 12, Ubuntu 10.04, Ubuntu 12.04, Ubuntu 12.10**
   - Rudder agent (binary packages available from
     ([Normation](http://www.normation.com)): **Windows Server 2008-2012,
     AIX 5-6-7**
 
+**Note** : rudder-agent-thin and rudder-server-relay have been removed from Debian 5,
```
Comment (jooooooon) [inline release-data/changelogs/changelog-4.1.md]: Please remember there are no spaces in front of punctuation marks in English.

Could you rephrase this is "As of Rudder 4.1, the rudder-agent-thin and rudder-server-relay packages are no longer available in ..."?

**Comment by peckpeck** [comment]
Commit modified

File: release-data/changelogs/changelog-4.1.md [hunk 3]
```diff
@@ -19,12 +19,16 @@ a quick and easy way to get an installation for testing.
...
+  - Rudder server and Rudder relay: **Debian 7, Debian 8, RHEL/CentOS 6, RHEL/CentOS 7
+    (64 bits), SLES 11, SLES 12, Ubuntu 14.04, Ubuntu 16.04**
   - Rudder agent: all of the above plus **Debian 5, Debian 6,
     RHEL/CentOS 3, RHEL/CentOS 5, CentOS 7 (32 bits), Fedora 18, SLES
     10, SLES 12, Ubuntu 10.04, Ubuntu 12.04, Ubuntu 12.10**
   - Rudder agent (binary packages available from
     ([Normation](http://www.normation.com)): **Windows Server 2008-2012,
     AIX 5-6-7**
 
+**Note**: As of Rudder 4.1, the rudder-agent-thin and rudder-server-relay packges are
```
Comment (jooooooon) [inline release-data/changelogs/changelog-4.1.md]: s/packges/packages/ (sorry!)

**Comment by peckpeck** [comment]
Commit modified
